package.json

```diff
@@ -21,9 +21,11 @@
   },
   "license": "AGPL-3.0",
   "dependencies": {
+    "@hotwired/turbo": "^7.0.0-rc.1",
```

Comment:
I can see that _Turbo is currently in beta._: you don't think that's could be a problem? (I've no problem to work with beta version, but I remember that you used to pay attention to this)